src/pages/edge/identity-for-edge-network/tabs/faq.md

```diff
@@ -97,7 +113,11 @@ class MobileApp : Application() {
         MobileCore.configureWithAppID(ENVIRONMENT_FILE_ID)
         // Register Adobe Experience Platform SDK extensions
         MobileCore.registerExtensions(
-            listOf(Edge.EXTENSION, Identity.EXTENSION)
+            listOf(
+                Edge.EXTENSION, 
+                com.adobe.marketing.mobile.edge.identity.Identity.EXTENSION,
+                com.adobe.marketing.mobile.Identity.EXTENSION
```

Comment:
can we list the direct identity first, then edge identity same as the Java example?

Reply:
I did the reverse. I updated this Kotlin example to have the Edge Identity extension listed first.